src/webpack.js

```diff
@@ -47,32 +47,39 @@ const ensureDirectoryExists = filePath => {
 
 /**
  * @desc Get meta data by read content file appIcon.png, dappConfig.json
- * @return {void}
+ * @return {object} meta data
  */
 const getDappMetaData = () => {
-  const appIconPath = path.join(process.cwd(), 'appIcon.png');
-  const pathDappConfig = path.join(process.cwd(), 'dappConfig.json');
-  const dappConfig = checkFileExistAndPromptError(pathDappConfig)
-    ? JSON.parse(fs.readFileSync(pathDappConfig, 'utf8'))
+  const pathPackageConfig = path.join(process.cwd(), 'package.json');
```

Comment:
you still don't need to construct the absolut path just use `./package.json`

Reply:
done